scss/style.scss

```diff
@@ -0,0 +1,930 @@
+@import url("https://fonts.googleapis.com/css2?family=Dela+Gothic+One&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Chivo:ital,wght@1,700&display=swap");
```

Comment:
why are you adding fonts in HTML and CSS both files?

Reply:
removed